.github/scripts/validate_config_sync.py

```
@@ -0,0 +1,220 @@
+import json
+import sys
+import os
+import subprocess # For git diff
+from typing import Set, List
```

Ruff (0.14.10)

`typing.Set` is deprecated, use `set` instead

(UP035)